rust/parquet/src/encodings/decoding.rs

```diff
@@ -384,36 +278,18 @@ impl<T: DataType> Decoder<T> for DictDecoder<T> {
 /// See [`RleValueEncoder`](crate::encoding::RleValueEncoder) for more information.
 pub struct RleValueDecoder<T: DataType> {
     values_left: usize,
-    decoder: Option<RleDecoder>,
+    decoder: RleDecoder,
```

Comment:
Performance Notes:

Small and minor this gives a 5% boost on performance as the RLE encoder can just be established before use without needing to constantly check if it has been initialised for every value.

Reply:
My results show a significant improvement in rle encoder and decoder, FWIW.